src/CRUD.java

```diff
@@ -36,6 +35,22 @@ void read(){
 	        }
 	    }
 	    
+	    /**
+	     * UPDATE method
+	     * Updates the value of the T object on the List
+	     */
+	    void update(T obj) throws Exception {
+	    	if(!list.isEmpty()) {
+	    		for(T t: list) {
+	    			if(t.getId() == obj.getId()) {
```

Comment:
I was thinking that maybe, instead of doing t.getId() == obj.getId(), we might do t.equals(obj) given that equals does that

Reply:
Done!